src/olympia/amo/middleware.py

```diff
@@ -324,3 +323,19 @@ def process_response(self, request, response):
             response['X-AMO-Request-ID'] = request.request_id
 
         return response
+
+
+class PinningRouterMiddleware(_PinningRouterMiddleware, MiddlewareMixin):
+    """Wrapper to allow old style Middleware to work with django 1.10+.
+    Will be unneeded once
+    https://github.com/jbalogh/django-multidb-router/pull/41 is merged and a
+    new release of django-multidb-router is available."""
+    pass
+
+
+class CorsMiddleware(_CorsMiddleware, MiddlewareMixin):
```

Comment:
I note this wouldn't be necessary either if https://github.com/mstriemer/django-cors-headers/pull/1#issuecomment-216825677 had happened.